optuna_dashboard/ts/components/GraphEdfs.tsx

```diff
@@ -0,0 +1,139 @@
+import * as plotly from "plotly.js-dist-min"
+import React, { FC, useEffect } from "react"
+import {
+  Grid,
+  FormControl,
+  FormLabel,
+  MenuItem,
+  Select,
+  Typography,
+  SelectChangeEvent,
+  useTheme,
+  Box,
+} from "@mui/material"
+import { plotlyDarkTemplate } from "./PlotlyDarkMode"
+import { Target, getFilteredTrials, useObjectiveTargets } from "../trialFilter"
+
+const plotDomId = "graph-edfs"
+
+interface EdfPlotInfo {
+  study_name: string
+  trials: Trial[]
+}
+
+export const GraphEdfs: FC<{
```

Comment:
Do you think we can unify this component with GraphEdf?

Reply:
Unified.